proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -235,7 +236,8 @@ public void start() throws Exception
             ResourceLeakDetector.setLevel( ResourceLeakDetector.Level.DISABLED ); // Eats performance
         }
 
-        eventLoops = PipelineUtils.newEventLoopGroup( 0, new ThreadFactoryBuilder().setNameFormat( "Netty IO Thread #%1$d" ).build() );
+        eventLoops = PipelineUtils.newEventLoopGroup( 0, new ThreadFactoryBuilder().setNameFormat( "Netty Boss IO Thread #%1$d" ).build() );
```

Comment:
Curious why we should have a separate event loop for boss threads, and having such a large one at that.